CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 * Silence the warning for missing mapping file on variants that don't enable minification (#111)
 * Bump: sentry-cli to 1.64.1
 * Add support for hashing sentry-cli to trigger a re-download upon bumps (#110)
+* Update the build matrix configuration to test against latest AGP/Gradle versions (#114)
```

Comment:
```suggestion
```

changelog is meant for end-users, as both are related to CI or upgrading sentry-cli within the project, we can use `#skip-changelog` under the PR description so Danger does not complain